.travis.yml

```diff
@@ -35,6 +35,11 @@ notifications:
       - "chat.freenode.net#oppia"
     on_success: never
     on_failure: always
+  webhooks:
+    urls:
+      - https://webhooks.gitter.im/e/f8f782497ec1ffcea2e7
```

Comment:
@seanlip This is the unique url of the webhook integrated with the Gitter channel.